patches/v8/BUILD.gn.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/BUILD.gn b/BUILD.gn
+index 83082271b8b76179b41e943190303e31fa639a95..2363c63c56598b785a9647ebcfaefde08f671067 100644
+--- a/BUILD.gn
++++ b/BUILD.gn
+@@ -643,6 +643,7 @@ config("internal_config_base") {
+   configs = [ ":v8_tracing_config" ]
+ 
+   include_dirs = [
++    "//brave/chromium_src/v8",  # v8-specific //brave/chromium_src support.
```

Comment:
`brave_chromium_src_support` is already in `default_compiler_configs` so can't we just add this to `include_dirs = [ "//brave/chromium_src" ]` ?

Reply:
> `brave_chromium_src_support` is already in `default_compiler_configs` so can't we just add this to `include_dirs = [ "//brave/chromium_src" ]` ?

This wouldn't work, because `//brave/chromium_src/v8` contains `src/` dir which will conflict with our upstream includes via `src/` prefix.

Reply:
can we do something like `include_dirs = [ "//brave/chromium_src/v8" ] + include_dirs`? Not sure if gn will complain or not, but then we could do it all on one line

Reply:
> can we do something like `include_dirs = [ "//brave/chromium_src/v8" ] + include_dirs`? Not sure if gn will complain or not, but then we could do it all on one line

yep. had to add another var to clear `include_dirs` before reassigning. thanks!